tests/MyGame/Example/Vec3.cs

```diff
@@ -6,21 +6,23 @@ namespace MyGame.Example
 using System;
 using FlatBuffers;
 
-public sealed class Vec3 : Struct {
-  public Vec3 __init(int _i, ByteBuffer _bb) { bb_pos = _i; bb = _bb; return this; }
+public struct Vec3 : Accessor
```

Comment:
Definitely consider calling it `IAccessor`, to be idiomatic - it'd hurt my eyes otherwise :)

Probably something like `IFlatbufferObject` if it's the base for all tables/structs.